src/test/java/uk/ac/ebi/eva/pipeline/jobs/steps/VepInputGeneratorStepTest.java

```diff
@@ -65,17 +79,25 @@ public void setUp() throws Exception {
 
     @Test
     public void shouldGenerateVepInput() throws Exception {
-        // TODO This test can't be changed to use temporary directory right now, as vepInput is a composite parameter
-        // that is not being recalculated at execution time.
         mongoRule.restoreDump(getResourceUrl(MONGO_DUMP), jobOptions.getDbName());
```

Comment:
This shouldn't use `JobOptions` any more but a random database based on the `TemporaryMongoRule`.